app/controllers/application_controller.rb

```diff
@@ -10,13 +10,29 @@ class ApplicationController < ActionController::Base
   helper LinksHelper
   helper_method :current_user
   helper_method :sso_identity
+  helper_method :available_estates
+  helper_method :current_estate
 
   def current_user
     sso_identity&.user
   end
 
   def current_estate
-    sso_identity&.current_estate
+    available_estates.find do |estate|
+      estate.sso_organisation_name == sso_identity&.current_organisation
+    end
+  end
+
+  def available_estates
+    @available_estates ||=
+      if sso_identity.
+         available_organisations.
+         include?(SignonIdentity::DIGITAL_ORG)
```

Comment:
Couldn't think of a better place to put the the `DIGITAL_ORG` constant, any ideas?